tracking/include/pcl/tracking/impl/tracker.hpp

```diff
@@ -1,39 +1,39 @@
 #ifndef PCL_TRACKING_IMPL_TRACKER_H_
 #define PCL_TRACKING_IMPL_TRACKER_H_
 
-#include <pcl/common/eigen.h>
 #include <ctime>
```

Comment:
`ctime` comes after the PCL headers

Reply:
Noted. Same pattern exists in other files too, I'll change them too.

Also, I couldn't find the documentation for the hierarchy, any link?

Reply:
None. We *should* put it in though.

Reply:
Done. #3904

Reply:
Done, for tracking. A more robust approach via .clang-format to be done in another PR.